test/unit/extract/test_mapping.py

```diff
@@ -0,0 +1,196 @@
+import os
+import json
+import pytest
+from unittest import mock, TestCase
+
+import fhirpipe.extract.mapping as mapping
+import fhirpipe.extract.graphql as gql
+
+from test.unit import exported_source, patient_pruned
+
+
+@mock.patch("fhirpipe.extract.mapping.get_mapping_from_file")
+@mock.patch("fhirpipe.extract.mapping.get_mapping_from_graphql")
+def test_get_mapping(mock_get_mapping_from_graphql, mock_get_mapping_from_file):
```

Comment:
can you use test classes ? example in fhir-api/tests/db/db_test.py

Reply:
I think that the convention with pytest is to simply have functions (instead of unittests's classes)

Reply:
didn't know that, works for me then